query-grammar/src/query_grammar.rs

```diff
@@ -1626,11 +1627,11 @@ mod test {
     fn test_exist_query() {
         test_parse_query_to_ast_helper("a:*", "\"a\":*");
         test_parse_query_to_ast_helper("a: *", "\"a\":*");
-        // an exist followed by default term being b
-        test_is_parse_err("a:*b", "(*\"a\":* *b)");
 
-        // this is a term query (not a phrase prefix)
+        // these are term/wildcard query (not a phrase prefix)
         test_parse_query_to_ast_helper("a:b*", "\"a\":b*");
```

Comment:
we should use a different way to serialize field exists query.

for instance `$field_exists("fieldname")`